layouts/partials/article.html

```diff
@@ -27,6 +27,17 @@
     {{ $content = len .Content }}
 {{ end }}
 
+<script>
+    function copyPermalink(link) {
+        // Copy the article link to the clipboard
+        let sanitizedLink = link;
+        while (sanitizedLink[0] === '/') {
+            sanitizedLink = sanitizedLink.substr(1);
```

Comment:
[String.prototype.substr()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr) is deprecated

Reply:
Thanks! Changed it.